src/redux/actions/projects/loadProjects.js

```diff
@@ -1,29 +1,83 @@
-import fetchAPI from '../../../utils/fetchAPI';
-import pushNotificationMessage from '../../../utils/pushNotificationMessage';
-import endUserMessages from '../../../utils/endUserMessages';
-import { isServerError, throwIfRequestFailed } from '../../../utils/fetchErrors';
-import { PROJECTS_ERROR, PROJECTS_LOADED, PROJECTS_LOADING } from '../../actionTypes/projects';
-import loadSamples from '../samples/loadSamples';
+import fetchAPI from 'utils/fetchAPI';
+import pushNotificationMessage from 'utils/pushNotificationMessage';
+import endUserMessages from 'utils/endUserMessages';
+import { isServerError, throwIfRequestFailed } from 'utils/fetchErrors';
+
+import config from 'config';
+import { api } from 'utils/constants';
+
+import { PROJECTS_ERROR, PROJECTS_LOADED, PROJECTS_LOADING } from 'redux/actionTypes/projects';
+import loadSamples from 'redux/actions/samples/loadSamples';
+
+const toApiV1 = (experimentListV2) => {
+  const projectsListV1 = experimentListV2.map((experimentData) => {
+    const {
+      createdAt,
+      updatedAt,
+      id,
+      samplesOrder,
+      ...restOfExperimentData
+    } = experimentData;
+
+    // Not assigning "lastAnalyzed" because that is no longer used anywhere
```

Comment:
isn't lastAnalysed used in data management to display the dates for `Modified` and `Created` for projects?
<img width="337" alt="Screenshot 2022-04-05 at 09 59 40" src="https://user-images.githubusercontent.com/65414651/161706937-263506af-25fc-4e77-b1a0-5dc4a00df97c.png">